src/Routes.js

```diff
@@ -109,7 +107,7 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/signing_official_console/researchers" component={SigningOfficialResearchers} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.signingOfficial]} />
     <AuthenticatedRoute path="/signing_official_console/dar_requests" component={SigningOfficialDarRequests} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.signingOfficial]} />
     <AuthenticatedRoute path="/signing_official_console/data_submitters" component={SigningOfficialDataSubmitters} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.signingOfficial]} />
-    <AuthenticatedRoute path="/signing_official_console" component={SigningOfficialConsole} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.signingOfficial]} />
+    {props.env === 'prod' ? <AuthenticatedRoute path="/signing_official_console" component={SigningOfficialConsole} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.signingOfficial]}/> : ''}
```

Comment:
This change disappears the SO console:

![Screen Shot 2022-06-29 at 5 18 26 PM](https://user-images.githubusercontent.com/116679/176546609-a0b79c2a-3aed-4067-bc58-b03345c6ca41.png)

Reply:
As mentioned in the PR description. the navigation updates in DUOS-1866 cover that issue

Reply:
This PR has to stand on its own. We can easily point both new and old paths to the new component and clean up unused components.

Reply:
Sorry, meant to continue that thought ... and clean up unused components in a follow on story (https://broadworkbench.atlassian.net/browse/DUOS-1823)